keps/sig-apps/3335-statefulset-slice/README.md

```diff
@@ -658,14 +671,14 @@ You can take a look at one potential example of such test in:
 https://github.com/kubernetes/kubernetes/pull/97058/files#diff-7826f7adbc1996a05ab52e3f5f02429e94b68ce6bce0dc534d1be636154fded3R246-R282
 -->
 
-Additional e2e tests will be added when targeting the Beta stage. These will
-validate the behavior of the cluster when enabled and disabled and ensure that
-existing behavior (eg: not specifying the new `ordinals.start` API) is
+Existing e2e tests will validate that when the feature is enabled, but not in
+use that the existing behavior (eg: not specifying the new `ordinals.start` API) is
 preserved.
 
-### Rollout, Upgrade and Rollback Planning
+Additionally unit tests for validating enablement/disablement will be added in
```

Comment:
We pretty much want the test like this mentioned in the comment to validate the field behavior.
```
You can take a look at one potential example of such test in:
https://github.com/kubernetes/kubernetes/pull/97058/files#diff-7826f7adbc1996a05ab52e3f5f02429e94b68ce6bce0dc534d1be636154fded3R246-R282
```

Reply:
Noted. Will add a unit test prior to updating the feature gate to beta (https://github.com/kubernetes/kubernetes/pull/115536).

These tests will mainly exercise the logic in strategy.go. I do have unit tests that validate this logic explicitly (https://github.com/kubernetes/kubernetes/pull/112744/files#diff-bfde8b1db7251cfd89bbce04b4397ea0b7579b7988ff4adbbb1725cbf06312c3R406), but I will add additional tests to validate this logic holistically when making changes for code freeze.

Reply:
Great thanks. Please link it here once it merges (I will be asking for it in GA PRR anyway :) )